lxd/instance_post.go

```diff
@@ -300,6 +300,23 @@ func instancePost(d *Daemon, r *http.Request) response.Response {
 		targetGroupName = after
 	}
 
+	// Unset "volatile.cluster.group" if the instance is manually moved to a cluster member.
+	if targetMemberInfo != nil && targetGroupName == "" {
+		err = s.DB.Cluster.Transaction(r.Context(), func(ctx context.Context, tx *db.ClusterTx) error {
+			if inst.LocalConfig()["volatile.cluster.group"] != "" {
```

Comment:
We should do this check before starting a transaction as it might not be needed.